python/llm/setup.py

```diff
@@ -322,7 +323,7 @@ def setup_package():
                         "xpu-2-0": xpu_20_requires,
                         "xpu-2-1": xpu_21_requires,
                         "serving": serving_requires,
-                        "llama_cpp": ["bigdl-core-cpp==" + VERSION + ";platform_system=='Linux'"]},
+                        "cpp": ["bigdl-core-cpp==" + VERSION + ";platform_system=='Linux'"]},
```

Comment:
does it work on windows?

Reply:
Let's do Linux first, but windows support is also needed

Reply:
Got it.